release/7-3/debian12/docker/Dockerfile

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# Docker image file that describes an Debian image with PowerShell installed from Microsoft APT Repo
+ARG hostRegistry=psdockercache.azurecr.io
+FROM ${hostRegistry}/debian:bookworm AS installer-env
+
+# Define Args for the needed to add the package
+ARG PS_VERSION=7.3.1
+ARG PS_PACKAGE=powershell-${PS_VERSION}-linux-x64.tar.gz
+ARG PS_PACKAGE_URL=https://github.com/PowerShell/PowerShell/releases/download/v${PS_VERSION}/${PS_PACKAGE}
+ARG PS_INSTALL_VERSION=7
+
+# Define the folder we will be installing PowerShell to.
+ENV PS_INSTALL_FOLDER=/opt/microsoft/powershell/${PS_INSTALL_VERSION}
+
+# Create the install folder.
+RUN mkdir -p ${PS_INSTALL_FOLDER}
+
+RUN --mount=type=cache,target=/var/lib/apt \
+    --mount=type=cache,target=/var/cache/apt \
+    apt-get update \
+    && apt-get install --no-install-recommends -y \
+    # curl is required to grab the Linux package
+        curl \
+        tar \
+    # less is required for help in powershell
+        less \
+    # requied to setup the locale
+        locales \
+    # required for SSL
+        ca-certificates \
+    # Download the Linux package and save it
+    && echo ${PS_PACKAGE_URL} \
+    && curl -sSL ${PS_PACKAGE_URL} -o /tmp/powershell.tar.gz
+
+# Unzip the linux powershell.tar.gz
+RUN tar zxf /tmp/powershell.tar.gz -C ${PS_INSTALL_FOLDER}
```

Comment:
does the deb file not work?

Reply:
We should prefer the deb files.

If it doesn't work, we need to also fix the package.